src/pretalx/orga/templates/orga/cfp/question_view.html

```diff
@@ -28,7 +29,7 @@
             <tr>
                 <th>{% translate "Question" %}</th>
                 <th>{% translate "Target" %}</th>
-                <th>{% translate "required" %}</th>
+                <th>{% translate "require" %}</th>
```

Comment:
Is this a necessary change? It forces all translations to update, so I'd like to be conservative if not.

Reply:
Fixed.